nservicebus/upgrades/6to7/index.md

```diff
@@ -211,4 +211,13 @@ WARNING: The `{AppDomain.CurrentDomain.BaseDirectory}\License\License.xml` path
 
 When running on the full .NET Framework, endpoints will [continue to search the registry locations](/nservicebus/licensing/) for a suitable license.
 
-When running on the .NET Core platform, endpoints **will not search the registry**, even when running on Windows.
\ No newline at end of file
+When running on the .NET Core platform, endpoints **will not search the registry**, even when running on Windows.
+
+
+## Connection Strings
+
```

Comment:
Should also mention that the implicit "NServiceBus/Transport" connection string should be avoided? (Or is the runtime WARN log enough?)

Reply:
I think mention it is a good idea